### PR TITLE
Suitep dashboard

### PR DESCRIPTION
**Pull request by daniel-samson**
Reworked the dashboard for SuiteP

## Description
I converted the dashboard to bootstrap tabs and added bootstrap modal dialog. Other changes such as the new icons in the title of each dashlet.

## Motivation and Context
Update for the new theme.

## How To Test This
Test each feature of the dashboard

## Types of changes
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: modules/Home/AddDashboardPages.php [hunk 1]
```diff
@@ -48,7 +48,7 @@
     $html .= "<table>";
     $html .= "<tr>";
     $html .= "<td><label for='dashName'>".$GLOBALS['app_strings']['LBL_ENTER_DASHBOARD_NAME']."</label></td>";
-    $html .= "<td><input name='dashName' id='dashName'/></td>";
+    $html .= "<td><input type=\"text\" name='dashName' id='dashName'/></td>";
```
Comment (samus-aran) [inline modules/Home/AddDashboardPages.php]: Why escape?  Just single quotes

File: modules/Home/Home.tpl [hunk 2]
```diff
@@ -78,6 +79,8 @@ var moduleName = 'Home';
 
 
 <div id="pageContainer">
+	cheese
```
Comment (samus-aran) [inline modules/Home/Home.tpl]: ?  Please - no Easter eggs

File: themes/SuiteR/include/MySugar/tpls/MySugar.tpl [hunk 3]
```diff
@@ -156,7 +154,7 @@
 </div>
 <script type="text/javascript" src="include/MySugar/javascript/AddRemoveDashboardPages.js"></script>
 <script type="text/javascript" src="include/MySugar/javascript/retrievePage.js"></script>
-<link rel="stylesheet" type="text/css" href="themes/SuiteP/css/dashboardstyle.css">
+<link rel="stylesheet" type="text/css" href="themes/SuiteR/css/dashboardstyle.css">
```
Comment (samus-aran) [inline themes/SuiteR/include/MySugar/tpls/MySugar.tpl]: Can we also include Suite7.

Comment (daniel-samson) [inline themes/SuiteR/include/MySugar/tpls/MySugar.tpl]: Suite7 doesn't use a custom MySmarty.tpl to customise the dashboard. It relies on the original MySugar.tpl found in include/MySugar/tpls/MySugar.tpl.

**Pull request merged**